documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -453,11 +454,14 @@ This is different from the units used for xref:con-common-configuration-resource
 
 .`-Xms` and `-Xmx` options
 
-Specify heap configuration for your components using the `-Xms` option to set an initial heap size and the `-Xmx` option to set a maximum heap size.
+In addition to setting memory request and limit values for your containers, you can use the `-Xms` and `-Xmx` JVM options to set specific heap sizes for your JVM.
+Use the `-Xms` option to set an initial heap size and the `-Xmx` option to set a maximum heap size.
 
-If you don't specify heap configuration, but configure memory request or limit, the Cluster Operator imposes default heap sizes automatically.
-
-The Cluster Operator sets maximum and minimum heap values based on a percentage allocation of available memory in the node.
+Specify heap size to have more control over the memory allocated to your JVM.
+Heap sizes should make the best use of a container's xref:con-common-configuration-resources-reference[memory limit (and request)] without exceeding it.
+Heap size and any other memory requirements need to fit within a specified memory limit. 
+If you don't specify heap size in your configuration, but you configure a memory limit (and request), the Cluster Operator imposes default heap sizes automatically.
+The Cluster Operator sets default maximum and minimum heap values based on a percentage allocation of available memory in the node where the container is running.
```

Comment:
This is not true. If the resources are set, we set the heap as a percentage of the resource configuration. But we never set it based on the node memory.

Reply:
Now this makes much more sense! The node memory wasn't picked up in the last review, so I assumed that was how it was. Changed to:

> The Cluster Operator sets default maximum and minimum heap values based on a percentage of the memory resource configuration.